ci/stress_tests/run_application_stress_tests.sh

```diff
@@ -12,11 +12,16 @@
 
 
 ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
-RAY_VERSION=$(git describe --tags --abbrev=0)
 RESULT_FILE=$ROOT_DIR/"results-$(date '+%Y-%m-%d_%H-%M-%S').log"
 
-echo "Testing on latest version of Ray: $RAY_VERSION"
 echo "Logging to" $RESULT_FILE
+echo -e $RAY_AWS_SSH_KEY > /root/.ssh/ray-autoscaler_us-west-2.pem && chmod 400 /root/.ssh/ray-autoscaler_us-west-2.pem || true
```

Comment:
If someone gets access to this key, is that an issue? They could just submit a PR and call `cat` on it I think.

Reply:
Well the env variable is only set when this is run, and this only is run on master, rather than any Jenkins build, so it should be fine